domain/robotouille.json

```diff
@@ -99,6 +103,46 @@
             "name": "iscut",
             "param_types": ["item"]
         },
+        {
+            "name": "ishotdog",
+            "param_types": ["item"]
+        },
+        {
+            "name": "isbun",
+            "param_types": ["item"]
+        },
+        {
+            "name": "isgrillable",
+            "param_types": ["item"]
+        },
+        {
+            "name": "istoastable",
+            "param_types": ["item"]
+        },
+        {
+            "name": "isgrilled",
+            "param_types": ["item"]
+        },
+        {
+            "name": "istoasted",
+            "param_types": ["item"]
+        },
+        {
+            "name": "has_condiments",
+            "param_types": ["item"]
+        },
+        {
+            "name": "has_ketchup",
+            "param_types": ["item"]
+        },
+        {
+            "name": "has_mustard",
+            "param_types": ["item"]
+        },
+        {
+            "name": "has_ketchup_mustard",
```

Comment:
if an item has both ketchup and mustard it should have the `has_ketchup` and `has_mustard` predicates.

Reply:
updated